keps/sig-architecture/20191023-conformance-without-beta.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Ensure Conformance Tests Do Not Require Beta APIs or Features
+authors:
+  - "@liggitt"
+owning-sig: sig-architecture
+participating-sigs:
+  - sig-testing
+  - sig-api-machinery
+reviewers:
+  - "@deads2k"
+  - "@bentheelder"
+  - "@timothysc"
+  - "@smarterclayton"
+  - "@johnbelamaric"
+approvers:
+  - "@timothysc"
+  - "@smarterclayton"
+  - "@johnbelamaric"
+creation-date: 2019-10-23
+last-updated: 2019-10-23
+status: provisional
```

Comment:
What's left to make this implementable?

Reply:
Not much, I just default new KEPs to provisional. I'd like an ack on the mechanism from @deads2k, and an approve from @timothysc